src/app/pages/datasets/components/dataset-form/dataset-form.component.spec.ts

```diff
@@ -66,6 +69,7 @@ describe('DatasetFormComponent', () => {
         EncryptionSectionComponent,
         QuotasSectionComponent,
         OtherOptionsSectionComponent,
+        IxModalHeaderComponent,
```

Comment:
It's better not to declare `IxModalHeaderComponent`s like this. Instead add `mockProvider(IxSlideInService2)` to providers.